docker/test-server/etebase-server.ini

```diff
@@ -3,10 +3,19 @@ secret_file = secret.txt
 debug = true
 static_root = /data/static
 media_root = /data/media
+AWS_ACCESS_KEY_ID = AKIARXDELOGPTJKCHC5B
```

Comment:
Should never hardcode credentials and commit that. Always should use env variables. https://www.twilio.com/blog/environment-variables-python

Reply:
Actually in this repo, .ini file is used to store all environment variables, if you see in settings.py file, you will notice that all the env variables are loaded from this config file (.ini)